templates/tfengine/recipes/devops.hcl

```diff
@@ -214,6 +220,7 @@ schema = {
         module.
       EOF
       type        = "boolean"
+      default     = "false"
```

Comment:
Shouldn't these defaults be booleans, not strings?

Reply:
Unfortunately we can't have arbitrary type values there. This is because it is parsed during `generate_schema_docs` using this [struct property](https://github.com/GoogleCloudPlatform/healthcare-data-protection-suite/blob/master/scripts/generate_schema_docs/schema.go#L34). I think the best we can do is define it as string and:

1. For booleans, just comment the boolean like `"false"`
2. For strings, escape quotes like in `"\"false\""`

Reply:
Oh, I see. Then it's fine, just keep them as strings.